combobulate-manipulation.el

```diff
@@ -1122,6 +1122,19 @@ buffer.
          (combobulate-skip-whitespace-forward))
        :reset-point-on-abort t))))
 
+(defun combobulate-envelop-region (template)
+  "Insert Combobulate TEMPLATE around active region."
+  (interactive)
+  (save-excursion
```

Comment:
@mickeynp 

Re 

> Point placements are disregarded when region mode is engaged.

When removing this `save-excursion` the point placement seems to work as expected. As I am not the most proficient in Emacs lisp I am not sure if this would be an sensible change. What do you think?

Also, how does the cursor positioning work in `combobulate-envelop-node` anyways as it also makes use of `save-excursion`?

Reply:
Well, I think removing it is probably okay, provided everything else continues to work. I suppose you could argue that, if there is no @ in the template at all, that saving the excursion is possibly useful?